base/src/main/java/uk/ac/ebi/atlas/solr/query/builders/BioentityIdentifierQueryBuilder.java

```diff
@@ -4,30 +4,42 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import uk.ac.ebi.atlas.search.SemanticQueryTerm;
-
-
-public class BioentityIdentifierQueryBuilder extends SolrQueryBuilder<BioentityIdentifierQueryBuilder>{
+import uk.ac.ebi.atlas.species.Species;
+
+/**
```

Comment:
Now that we have a better domain model (instead of stringly typed species and query term like they used to be) possibly it's time to ditch that builder. They're really Java's way of doing named or optional parameters (because constructors with, say, ten string parameters, are unmaintainable) but since this takes a Species and a SemanticQueryTerm there's no danger of switching them around (and we're handling allowing no species with a "Null object pattern" of unknown species).